discord/abc.py

```diff
@@ -1605,6 +1605,8 @@ async def send(
                 )
 
             components = view.to_components()
+            if view.is_components_v2():
+                flags.is_components_v2 = True
```

Comment:
We should add an error here if `content` or `embeds` are provided, because some of the discord errors aren't very clear